packages/core/src/App/Containers/CurrencySelectionModal/curreny-selection-modal.tsx

```diff
@@ -103,7 +110,11 @@ const CurrencySelectionModal = ({
                 <Button
                     className='block-button'
                     onClick={() => {
-                        setTimeout(() => openRealAccountSignup('manage'), 500);
+                        setTimeout(() => {
```

Comment:
@thisyahlen-deriv I think it would be safer if we clear the timeout on the next click 🤔

Reply:
Done